ci/unit/test_nlt_node.sh

```diff
@@ -20,4 +20,4 @@ cd build
 sudo bash -c ". ./utils/sl/setup_local.sh; ./utils/setup_daos_admin.sh"
 
 # NLT will mount /mnt/daos itself.
-./utils/node_local_test.py --max-log-size 400MiB --perf-check all
+./utils/node_local_test.py --max-log-size 400MiB --dfuse-dir /localhome/jenkins/ --perf-check all
```

Comment:
Not sure if the 80-chars warning is still relevant, but an easy fix for that if it is:
```suggestion
./utils/node_local_test.py --max-log-size 400MiB --dfuse-dir /localhome/jenkins/ \
                           --perf-check all
```